### PR TITLE
modify: LinkMovementMethod makes list item unclickable

### PR DESCRIPTION
**Pull request by ohmerhe**
I solve this problem refer to the answer in stackoverflow. http://stackoverflow.com/questions/8558732/listview-textview-with-linkmovementmethod-makes-list-item-unclickable

### CODE REVIEW

**Comment by dschuermann** [comment]
Thanks for your pull request, please fix the annotated lines.

**Comment by ohmerhe** [comment]
I am sorry for my carelessness.

File: src/main/java/org/sufficientlysecure/htmltextview/LocalLinkMovementMethod.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import android.text.Layout;
+import android.text.Selection;
+import android.text.Spannable;
+import android.text.method.LinkMovementMethod;
+import android.text.method.Touch;
+import android.text.style.ClickableSpan;
+import android.view.MotionEvent;
+import android.widget.TextView;
+
+/**
```
Comment (dschuermann) [inline src/main/java/org/sufficientlysecure/htmltextview/LocalLinkMovementMethod.java]: Remove this comment and add a proper Apache license comment above the whole file like in the other files of this library

**Comment by dschuermann** [comment]
No problem, that's normal for pull requests.
There is just one thing missing, I annotated it again.

**Comment by ohmerhe** [comment]
dschuermann, is there any problem?

**Comment by dschuermann** [comment]
thanks!

**Pull request merged**